ONSdigital/eq-submission-confirmation-consumer
# Remove pipenv lock deprecated warning

### What is the context of this PR?

When running `pipenv lock -r > requirements.txt` via `make install` there is a deprecation warning. This change will fix that and remove the warning. But there will be a change to the format of the requirements.txt file.

### How to review
- run `make install` on master and on this branch
- see the difference in the format of the requirements.txt file created before and after the change
- also check that the deprecated warning is removed

### Checklist

\*[ ] Tests updated
